launch.sh

```
@@ -3,16 +3,21 @@
 echo "🚀 Launching Dhaher Trading Plan AI..."
 echo "----------------------------------------"
 
+# Load environment variables
+if [ -f .env ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)